site/en/blog/compression-streams-api/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Compression and decompression in the browser with the Compression Streams API
+subhead: >
+  Write smaller web apps that don't need to ship their own compression or decompression library
+date: 2022-08-26
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/koW7cNlWa6BUlJsQR4wI.jpg
+alt: Clamp to symbolize how data gets compressed.
+author: thomassteiner
+tags:
+  - capabilities
+---
+
+The [Compression Streams API](https://developer.mozilla.org/docs/Web/API/Compression_Streams_API)
+provides a JavaScript API for compressing and decompressing streams of data using the gzip or
+deflate (or deflate-raw) formats.
+
+With built in compression JavaScript applications do not need to include a compression
+library, making the download size of the application smaller. Chrome and Safari Technology Preview now
```

Comment:
```suggestion
library, making the download size of the application smaller. Stable Chrome and Safari Technology Preview now
```

Reply:
As originally written, I took this to mean that the feature was in beta or canary. The other option was to list Safari first.